sources/Application/Audio/AudioFileStreamer.cpp

```diff
@@ -28,6 +28,8 @@ void AudioFileStreamer::Stop() {
   Trace::Debug("Streaming stopped");
 };
 
+bool AudioFileStreamer::IsPlaying() { return mode_ == AFSM_PLAYING; }
```

Comment:
Are we setting the mode or returning if it's playing or not?

Reply:
Ah good point, its just returning true if its currently playing.  I'll put parens around the equals operator to make it clear .